src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -45,6 +45,15 @@ public JsonRpcTests(ITestOutputHelper logger)
         this.clientRpc = JsonRpc.Attach(this.clientStream);
     }
 
+    private interface IServer
+    {
+        [JsonRpcMethod("AnotherName")]
+        string AManBy(string name);
```

Comment:
nit: I would make it a "ARoseBy" :)

Reply:
Two quick thoughts: 1) don't see a test for invoking a member by it's non-attributed name, and the expected result (though could have missed it), and 2) didn't see a test for invoking an async method w/o async suffix for validating candidatemap

Reply:
Thanks. I guess I exposed a weakness in my would-be cultured education. I've fixed it to `ARoseBy`.

Reply:
There is indeed a negative test for calling a method by its CLR name. Here it is, with the correction you called for:

```cs
        await Assert.ThrowsAsync<RemoteMethodNotFoundException>(() => this.clientRpc.InvokeAsync(nameof(IServer.ARoseBy), "andrew"));
```